katello/katello-backup

```diff
@@ -262,39 +324,42 @@ if @dir.nil?
   puts "**** ERROR: Please specify an export directory ****"
   puts optparse
   exit(-1)
-else
-  puts "Starting backup: #{Time.now}"
-  create_directories(@dir.dup)
-  FileUtils.cd @dir
-  generate_metadata
-  FileUtils.cp Dir.glob(File.join(@options[:incremental], '.*.snar')), @dir if @options[:incremental]
-  backup_config_files
+end
 
-  if @options[:logical_backup]
-    @databases.each do |database|
-      online_backup(database)
-    end
-  end
+puts "Starting backup: #{Time.now}"
+create_directories(@dir.dup)
+FileUtils.cd @dir
 
-  if @options[:snapshot]
-    snapshot_backup
-    compress_files
-  elsif @options[:online]
-    @databases.each do |database|
-      online_backup(database)
-    end
-  else
-    FileUtils.cd("/")
-    `katello-service stop #{EXCLUDED}`
-    @databases.each do |database|
-      offline_backup(database)
-    end
-    `katello-service start #{EXCLUDED}`
-    FileUtils.cd @dir
-    compress_files
+CONFIGS = configure_configs
+
+puts "Generating metadata ... "
+generate_metadata
+FileUtils.cp Dir.glob(File.join(@options[:incremental], '.*.snar')), @dir if @options[:incremental]
```

Comment:
I think this could benefit readability wise by wrapping the parameters in parens.